patterns/disposer.md

```diff
@@ -0,0 +1,84 @@
+# The Promise Disposer Pattern
```

Comment:
Is this really a pattern? Can you provide some sources? (articles, examples of code, etc)